app/components/participation-list/modal.js

```diff
@@ -39,6 +39,10 @@ export default class ParticipationListModalComponent extends Component {
     this.generateSelected();
   }
 
+  get selectedMandatees() {
+    return this.selectedMandateesMap;
```

Comment:
Hmm, that's not really what I meant with "convert selectedMandatees to a getter". The only reason this doesn't show the deprecation is [because it's not using a tracked map anymore](https://github.com/lblod/frontend-gelinkt-notuleren/pull/106/files#diff-f486ab0c87e76622c197ad111f9c13eea70c89eeea697ecf3065fbb15104b10eR33). If the map didn't need to be tracked in any way, just simply changing it like this would be enough to remove the deprecation:

`selectedMandatees = tracked(Map);`
to
`selectedMandatees = new Map();`

That's essentially what your new version does, only with an unneeded getter layer added on top.

What I meant was remove the selectedMandatees field on the class. Rename the `generateSelected` method selectedMandatees and convert it to a getter. Let that method create a local map variable and return that at the end. It would recompute if one of the arguments changes as well.

But like I said, if tracking isn't needed, simply removing tracked from the selectedMandatees map would be enough to stop the deprecation from showing.

Reply:
I tested and ir worked without problems, I will retest because the tracking should be needed

Reply:
Yeah, tracking it's not needed, so much simpler